### ISSUE TITLE
Hard to track plots with multiple experiments

### ISSUE THREAD

**Issue by dberenbaum** [iterative/vscode-dvc#2726]
It's hard for me to tell which line on a plot refers to which experiment:

- The experiments legend ribbon at the top of the plots view needs to be pinned so it’s visible even when scrolling down to see more plots.
- The colors are hard to parse. This is pretty much a repeat of https://github.com/iterative/vscode-dvc/issues/1757#issuecomment-1258619057.
- Renaming experiments and showing the experiments table next to the plots view could also help identify experiments more easily.

**Comment by shcheklein** [comment]
> Renaming experiments and showing the experiments table next to the plots view could also help identify experiments more easily.

Duplicate / Related #2457 ?

> The experiments legend ribbon at the top of the plots view needs to be pinned so it’s visible even when scrolling down to see more plots.

agreed. Let's make a separate ticket for this?

> The colors are hard to parse. This is pretty much a repeat of https://github.com/iterative/vscode-dvc/issues/1757#issuecomment-1258619057.

Let's also make a separate ticket?

**Comment by dberenbaum** [comment]
Put everything into other tickets.
